storybook/stories/foundations/design-language/elevation.mdx

```diff
@@ -0,0 +1,47 @@
+import { Meta, Typeset } from '@storybook/addon-docs/blocks';
+import { linkTo, hrefTo } from '@storybook/addon-links';
+
+<Meta title="Foundations/Design Language/Elevation" name="page" />
+
+# Elevation
+
+Elevation, through the use of shadows, visually indicates the layers where overlapping UI elements reside. Shadows create the illusion of depth, showing how one UI element is positioned above another on the z-axis. Elevation should be used to organize elements, establish a clear hierarchy, and draw focus to key components.
+
+In the WordPress components package there are four levels of elevation:
```

Comment:
Elevation (like every other tokens) is not part of the components package, but rather of the design system as a whole. Even technically, the elevation tokens are currently in `@wordpress/base-styles` — the components provided by the `@wordpress/components` package consume those tokens and therefore follow the design system.

In other words, from my perspective, the `@wordpress/components` package is a part of the design system's offering, as much as the token themselves.